lib/showoff.rb

```diff
@@ -523,6 +525,25 @@ def process_content_for_section_tags(content, name = nil, opts = {})
       doc.to_html
     end
 
+    # TODO: damn, this one is bad. This moves the notes div outside of the .content div, then removes
```

Comment:
My intent is actually for it to be that dumping ground for the stuff that won't fit anywhere else until we get the DOM manipulation cleaned up, and then for this method to be obsoleted. I'll add a note explaining that.